packages/@okta/vuepress-theme-prose/assets/css/components/_header.scss

```diff
@@ -265,7 +265,7 @@ header.page-header .link:focus {
 
   header.page-header .menu--slideout {
     position: fixed;
-    top: 90px;
+    top: 130px;
```

Comment:
double check in UI

Reply:
This was a fix for partially hidden search bar in mobile view.